### ISSUE TITLE
How to handle low quality reads

### ISSUE THREAD

**Issue by RyanBio** [marbl/canu#240]
Hi, everyone
I download some pacbio data from the ncbi for test, but the result was not ideal. The genome size was 39,832,981 bp, and the N50 was 214,426 bp. Total 18,321,842,183 bases (407.15 times coverage) was used, and the canu parameters were set 
```
genomeSize=45m \
corOutCoverage=80 \
errorRate=0.2 \
```
The message about reads after error correction was "Found 1,234,285,235 bases (27.42 times coverage)". 
I think there may be too many low quality reads extracted from the sra data.  The command was "fastq-dump --table SEQUENCE SRR*.sra "

So i hope to get help about:
1. Can i directly use the data extracted from the sra file for assemble ?
2. Can canu handle the extracted data which may be too many low quality reads ?

Thanks
Ryan

**Comment by brianwalenz** [comment]
What is the data set?  It might be useful to add to our test suite.

In general, don't set errorRate.  This is the error rate in _corrected_ reads.  I don't see anything in the code that uses error rate for correcting reads, but still, rerun without setting errorRate.

**Comment by RyanBio** [comment]
Hi, brain
Thanks for your quick reply.
I rerun without setting errorRate, but the result was similar. The errorRate was default set 0.025,
`-- Final error rates before starting pipeline:
--   
--   genomeSize          -- 40000000
--   errorRate           -- 0.025
--   
--   corOvlErrorRate     -- 0.075
--   obtOvlErrorRate     -- 0.075
--   utgOvlErrorRate     -- 0.075
--   
--   obtErrorRate        -- 0.075
--   
--   utgGraphErrorRate   -- 0.05
--   utgBubbleErrorRate  -- 0.0625
--   utgMergeErrorRate   -- 0.0375
--   utgRepeatErrorRate  -- 0.05
--   
--   cnsErrorRate        -- 0.0625
--
`
By the way, the canu version i used is v1.2.  Did the v1.3 version  change this parameter?

The test data i used was from https://www.ncbi.nlm.nih.gov/sra/SRX533606, from SRR1284075 to SRR1284945. The species is Neurospora crassa T1.

Best,
Ryan

**Comment by brianwalenz** [comment]
Most definitely update to 1.3.  There were significant improvements since 1.2.

**Comment by brianwalenz** [comment]
Running with the latest code and default parameters, I get:  98,199 corrected reads, of total length 1,136,011,800bp.  The final assembly is in 9 contigs.

Reopen or make a new issue if needed.
